pkg/grub/rootfs.cfg

```diff
@@ -349,6 +349,11 @@ function set_arm64_baremetal {
       set_global dom0_console "console=ttyAMA0,115200 console=ttyTCU0,115200 console=tty1 earlycon=tty1"
       set_global dom0_platform_tweaks "fbcon=map:0 video=efifb:off nospectre_bhb bl_prof_dataptr=2031616@0x271E10000 bl_prof_ro_ptr=65536@0x271E00000 nvgpu.devfreq_timer=delayed"
    fi
+   # Aetina AIX-PX11 Device (based on Jetson Orin AGX)
+   if [ "$smb_product" = "NVIDIA Jetson AGX Orin Developer Kit" ]; then
+      set_global dom0_console "console=ttyTCU0,115200 console=ttyAMA0,115200 console=tty1"
+      set_global dom0_platform_tweaks "fbcon=map:0 nospectre_bhb video=efifb:off bl_prof_dataptr=2031616@0x102C610000 bl_prof_ro_ptr=65536@0x102C600000 nvethernet.macsec_enable=0 eve_install_disk=mmcblk0 eve_persist_disk=nvme0n1"
```

Comment:
I'm curious: why do we sometimes redefine the `eve_install_disk` and `eve_persist_disk` args and sometimes not?

Reply:
We redefine these variables only for devices that have eMMC + NVMe (or other storage device), so we can install EVE in the eMMC and setup the persist storage in the other device....